llvm/lib/Target/RISCV/RISCV.td

```diff
@@ -190,11 +190,19 @@ def HasExtXCoreVMac
     : Predicate<"Subtarget->hasExtXCoreVMac()">,
                AssemblerPredicate<(any_of FeatureExtXCoreVMac),
                "'Xcorevmac' (Multiply-Accumulate)">;
+// CORE-V Multiply-Accumulate Extension
+def FeatureExtXCoreVMem
+    : SubtargetFeature<"xcorevmem", "HasExtXCoreVMem", "true",
```

Comment:
In the GCC compiler this feature is/will be called `xcorevpostinc`: https://github.com/openhwgroup/corev-binutils-gdb/pull/29

@jessicamills is this feature name set in stone? With the risk of starting bikeshedding over the feature name: I prefer the name `xcorevmem` for this feature, except it is already documented somewhere, that it is called `xcorevpostinc`.

Reply:
I don't believe these have been standardized anywhere. I prefer `Xcorevmem`, since this extension is more than just postinc. I suggest you discuss on OpenHW mattermost, and if necessary provide a pull request to the architecture manual (I'm not sure they document extension names though).

Reply:
The documentation doesn't specify the extension names, except the top-level `xcorev` name. So we should be free to choose, but also stay consistent.

Reply:
@SerkanMuhcu I verified with the GCC team and we're going with `xcorevmem`, so nothing to change for you here.